docs-library/10x-FY18-Project-Results.md

```diff
@@ -0,0 +1,41 @@
+
+**Notification Services** — Ended at this phase. [Final presentation here](https://docs.google.com/presentation/d/1mMtV_6CLa7wTx3XTBD7DoUhl-1jNh3Rk0tNBSKHzHoY/edit). NaaS was not selected for further funding. The PAB saw promise in the idea, but there were underlying concerns that led them to the decision against proceeding to Phase Three. The major concern was that TTS is not yet ready to take on another large-scale project with high up-front costs when we currently have both Login and cloud.gov struggling to become cost recoverable. A further concern was the ability of TTS to sustain a dedicated team for this project in the long-term. The PAB is open to considering NaaS again at some point in the future.
+
+There are still a lot of lessons to be learned from the great work the team undertook. The PAB expressed interest in revisiting this project in the future — perhaps a year from now if the staffing and financial situation at TTS has changed.
+
+**Federal Grantee Reporting** — Continued to next phase. [Final presentation here](https://docs.google.com/presentation/d/1w1N7bTz0fQ8e8MePlY3t6T450eYBlBVweEdZIQQ3rkI/edit#slide=id.g3770e36ff6_0_0). [README](https://docs.google.com/document/d/1wPUrxa6gfxdDVDQu4Z8xduz4TYo_EfwuZg5CJ_Ex4cA/edit). Decision was made to continue this project at the $650k Development phase. The PAB believes this project has “transformative potential” and was encouraged by Phase 2 results. They have advised the project team to consider the hand-off of product ownership and get OMB involvement early on in Phase 3.
+
+**U.S. Data Federation** — Continued to next phase. [Final presentation here](https://docs.google.com/presentation/d/1nPdhy0EnYXKFgm8aMfjwFSKu5kcxkjCai52uRXzOCfo/edit#slide=id.g3d2d046014_7_0). [README](https://docs.google.com/document/d/1oVVENSZxRC8r7eJyIqr72ajss4DyaAVvRyH_ZpYY04c/edit). Decision was made to continue this project at the $650k Development phase. 10x is funding this project because federated data efforts are increasingly seen as an engine for transparency, economic growth, and accountability, yet collecting that data remains a challenge. Despite the fact that efforts of this sort are increasing in frequency, agencies often have to build new efforts from scratch. Investment in reusable tools and approaches will streamline federated data efforts in the years to come. The PAB has requested regular check-ins with the project team going forward to ensure that implementation and partner milestones are proceeding as planned.
+
+## Phase Three. Development
```

Comment:
If there was nothing in phase 3 maybe we should leave a note under the heading that says so?